src/components/ErrorNotifications/ErrorNotifications.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect } from 'react';
+import classNames from 'classnames';
+import { Error } from '../../types/Error';
+
+type Props = {
+  error: Error,
+  handleErrors: (value: Error) => void;
+  removeError: () => void;
+};
+
+export const ErrorNotifications: React.FC<Props> = (
+  {
+    error,
+    handleErrors,
+    removeError,
+  },
+) => {
+  const handleCloseButtonClick = () => {
+    handleErrors(Error.NONE);
+  };
+
+  useEffect(() => {
+    removeError();
+  }, [error]);
+
+  return (
+    <div className={classNames(
+      'notification',
+      'is-danger',
+      'is-light',
+      'has-text-weight-normal',
+      {
+        hidden: error === Error.NONE,
+      },
+    )}
+    >
+      <button
+        type="button"
+        className="delete"
+        aria-label="Delete"
+        onClick={() => handleCloseButtonClick()}
+      />
+
+      {/* show only one message at a time */}
+      {error === Error.ONLOAD && (
+        'Unable to load todos'
+      )}
+
+      {error === Error.EMPTY && (
+        "Title can't be empty"
+      )}
+
+      {error === Error.ONADD && (
+        'Unable to add a todo'
+      )}
+
+      {error === Error.ONDELETE && (
+        'Unable to delete a todo'
+      )}
+
+      {error === Error.ONUPDATE && (
+        'Unable to update a todo'
+      )}
```

Comment:
Maybe you can use your enum and set an error message for all these cases, and after use just {error}

Reply:
Thank you for reviewing! I fixed toggleAll problem. But I am not sure that it's a good idea to store error sentences in an enum. If I got it right enum values are usually short and used for example for sorting purposes.

Reply:
agree, storing text in enum isn't a good idea